tests/fixtures/fixtures_models.py

```diff
@@ -0,0 +1,160 @@
+"""
+Fixtures for commonly used test models
+"""
+
+import pytest
+import pymc4 as pm
+import numpy as np
+import tensorflow as tf
+from pymc4 import distributions as dist
+import itertools
+
+@pytest.fixture(scope="function")
+def simple_model():
+    @pm.model()
+    def simple_model():
+        norm = yield pm.Normal("norm", 0, 1)
+        return norm
+
+    return simple_model
+
+# previously called simple_model in test_executor 
+# main difference is no @pm.model() and we use dist.Normal rather than pm.Normal
+@pytest.fixture(scope="function")
+def simple_model_dist():
+    def simple_model():
+        norm = yield dist.Normal("norm", 0, 1)
+        return norm
+
+    return simple_model
+
+@pytest.fixture(scope="module")
+def simple_model_class():
+    class ClassModel:
+        @pm.model(method=True)
+        def class_model_method(self):
+            norm = yield pm.Normal("norm", 0, 1)
+            return norm
+
+    return ClassModel()    
+
+@pytest.fixture(scope="function")
+def simple_model_with_deterministic(simple_model):
+    @pm.model()
+    def simple_model_with_deterministic():
+        norm = yield simple_model()
+        determ = yield pm.Deterministic("determ", norm * 2)
+        return determ
+
+    return simple_model_with_deterministic
+
+
+@pytest.fixture(scope="function")
+def simple_model_no_free_rvs():
+    @pm.model()
+    def simple_model_no_free_rvs():
+        norm = yield pm.Normal("norm", 0, 1, observed=1)
+        return norm
+
+    return simple_model_no_free_rvs
+
+
+# Previously called simple_model in tests_variational
+# Suggest this could be called something better?
```

Comment:
Request for help

Reply:
I think it will a good idea to rename that `simple_model` fixture to `conjugate_normal_model`.